src/TerraformDestroyCommand.groovy

```diff
@@ -0,0 +1,88 @@
+class TerraformDestroyCommand {
+    private boolean input = false
```

Comment:
I can't say yet whether or not it might make sense to say `class TerraformDestoryCommand extends TerraformApplyCommand`?